### PR TITLE
Added description and code to embed the form

### PR DESCRIPTION
**Pull request by renbot-bio**
I'm not sure I've done the embedding right @jfy133 and @mashehu :-/

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]
Looks good to me.

Best to ask @mashehu abut embedding sutff

File: markdown/events/2021/hackathon-october-2021.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-but in the mean time - book the dates in your calendar!
+With the recent release of the new nf-core template for Nextflow's DSL2, in this event we will be going full-steam ahead with conversion of nf-core pipelines from DSL1 to DSL2. The main objectives of this hackathon will be adding nf-core modules and rewriting existing pipelines to the new Nextflow language format.
+
+For Nextflow novices, we will be looking for help in writing Nextflow modules of commonly used bioinformatics tools.
+
+For more experienced developers we hope for contributions to writing subworkflows and pipeline code.
+
+In the spirit of collaboration and knowledge sharing, we plan to pair up both novices and experienced developers in pair programming to help get newcomers up to speed on writing and contributing nf-core modules through a pair-programming session for a limited number of participants. Make sure to sign up by September 27, 2021 to participate in pair-programming!
+
+This event is entirely virtual and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:. Do get in touch with us if you'd like to help us out!
```
Comment (mashehu) [inline markdown/events/2021/hackathon-october-2021.md:22]:
```suggestion
This event is entirely virtual and we are attempting to support people in all timezones :earth_americas: :earth_asia: :earth_africa:.
```
This is already taken care of in the form, which is probably also the best way to get in touch with us about it, besides slack. I think we should either mention how they can get in touch with us here or leave it out.

File: markdown/events/2021/hackathon-october-2021.md [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
+
+Sign up below!
```
Comment (mashehu) [inline markdown/events/2021/hackathon-october-2021.md:24]:
```suggestion
Sign up here 👇
```

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
Embed is working now, after #860.
Screenshot 2021-09-01 at 17 08 18

**Review by jfy133 — APPROVED** [review]

**Pull request merged**